packages/framework-core/src/booster-graphql-dispatcher.ts

```diff
@@ -59,12 +54,33 @@ export class BoosterGraphQLDispatcher {
     }
   }
 
+  private async verifyTokenFromEnvelop(
+    envelope: GraphQLRequestEnvelope
+  ): Promise<GraphQLRequestEnvelope | GraphQLRequestEnvelopeError> {
+    if ('token' in envelope && envelope.token) {
```

Comment:
Why not
```suggestion
    if (envelope.token) {
```

Reply:
This was to distinguish between types, but I think isn't needed

Reply:
Done.